Jenkinsfile

```diff
@@ -4,9 +4,9 @@
 */
 buildPlugin(
   timeout: 120, // reduce test failures due to timeout
+  forkCount: '1C', // run this number of tests in parallel for faster feedback.  If the number terminates with a 'C', the value will be multiplied by the number of available CPU cores
```

Comment:
Not sure if this is safe offhand. This plugin has highly memory-intensive tests and I am not sure if our agents can handle them (even more so given that our Windows agents are in a temporary situation right now).

Reply:
Would there be a way to give it a try though? If it helps reduce the runtime of tests and the agents can handle it, everyone wins.

Reply:
> Would there be a way to give it a try though?

I think running the tests a few times (a dozen at most) against the final CI infrastructure would be enough to give me confidence that this isn't introducing new flakiness. Right now the Windows CI infrastructure is temporary, not final, so I would want to wait until it is finalized.

Reply:
Sounds reasonable. Let’s wait for the Windows CI to recover and come back to this 👍🏼

Reply:
https://github.com/jenkins-infra/helpdesk/issues/4490 has been resolved and windows builds should be back to normal. Should we give this a try?

Reply:
@strangelookingnerd I ran two builds of this PR today:

https://ci.jenkins.io/job/Plugins/job/pipeline-model-definition-plugin/job/PR-752/3/
https://ci.jenkins.io/job/Plugins/job/pipeline-model-definition-plugin/job/PR-752/3/

These took about 1 hour and 45 minutes, while the same builds today on the default branch took only an hour:

https://ci.jenkins.io/job/Plugins/job/pipeline-model-definition-plugin/job/master/625/
https://ci.jenkins.io/job/Plugins/job/pipeline-model-definition-plugin/job/master/626/

I'm not sure why enabling parallelism slowed down the total test runtime, but looks like it did...

Reply:
It seems to only affect the Windows build. Linux build runtime is unchanged, give or take a couple minutes. But overall it does not seems worth it. Closing this PR.